generate_avocado_stubs.py

```
@@ -0,0 +1,180 @@
+def _collect_function_identifiers(tree: Tree) -> list[Node]:
+    """Return the identifier nodes parsed from function definitions and declarations.
+
+    Args:
+        tree (Tree): The tree from which to start collecting identifiers.
+
+    Returns:
+        list[Node]: The identifier nodes parsed from function definitions and declarations.
+    """
+    result = []
+
+    def traverse(node):
```

Ruff (0.15.2)

[warning] Missing return type annotation for private function `traverse`

Add return type annotation: `None`

(ANN202)